web-registry/src/pages/MyEcocredits/MyEcocredits.tsx

```diff
@@ -60,9 +72,15 @@ export const MyEcocredits = (): JSX.Element => {
   const [basketTakeTokens, setBasketTakeTokens] = useState<
     BasketTokens | undefined
   >(undefined);
+  const [txModalHeader, setTxModalHeader] = useState<string | undefined>(
+    undefined,
```

Comment:
doesn't the initialization default to `undefined` in useState hooks? maybe it can be omitted here, but I don't have a strong preference on it either (mostly to avoid line break maybe)